src/utils.js

```diff
@@ -145,6 +145,9 @@ const handleFormUpdate = function(e) {
         case 'textarea':
             obj[id] = e.target.value;
             break;
+        case 'button':
+            obj[id] = Number(e.target.value).toFixed(2)
```

Comment:
`toFixed()` returns a string representation of a float (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed) - so this is the string problem here. All we need to do to the number here is make sure it's of a `number` type, not do any formatting for display.

I'll make some changes on this branch.